### PR TITLE
Reverting build.xml to build static libs and adding missing schemes

### PR DESCRIPTION
**Pull request by bhariharan**

### CODE REVIEW

**Comment by bhariharan** [comment]
I added ```FMDB``` and ```CocoaLumberjack```, and removed ```SalesforceReact```, since we don't need it.

**Comment by khawkins** [comment]
Seems okay to me....

File: build/build.xml [hunk 1]
```diff
@@ -55,7 +81,7 @@
     <target name="buildSalesforceHybridSDK">
         <antcall target="buildLib">
             <param name="projectName" value="SalesforceHybridSDK" />
-            <param name="libName" value="SalesforceHybridSDK.framework" />
+            <param name="libName" value="libSalesforceHybridSDK.a" />
             <param name="headerDir" value="Headers/SalesforceHybridSDK" />
             <param name="schemeName" value="SalesforceHybridSDK" />
```
Comment (wmathurin) [inline build/build.xml]: The scheme name is probably going to be SalesforceHybridSDK-Static (or whatever we decide to call it)

Comment (bhariharan) [inline build/build.xml]: Yep - that's for the next round, when I bring in the static targets.

**Pull request merged**